graylog2-web-interface/package.json

```diff
@@ -19,6 +19,8 @@
     "build:analyze": "yarn build:stats && webpack-bundle-analyzer ./webpack/stats-$(git rev-parse --abbrev-ref HEAD).json ./target/web/build --mode=\"static\" --report=\"./webpack/analysis-$(git rev-parse --abbrev-ref HEAD).html\"",
     "lint": "eslint --ext js,jsx src",
     "lint:path": "eslint --ext js,jsx",
+    "lint:styles": "stylelint './src/**/*.{css,js,jsx,tsx}'",
```

Comment:
One of the last commits added checks on `css` but results don't seem right, all I get when running the command is ` 1    ✖  Unexpected token (1:0)` for each `css` file.
In my opinion we should either look why that happens and ensure it works, or leave `css` files out of the styling for now and take care of them in a separate PR, as we discussed some days ago.

Reply:
Will leave out as previously discussed and we can review this issue in the future (if any CSS files exist) 😁